go/oasis-test-runner/oasis/oasis.go

```diff
@@ -360,6 +365,27 @@ func (net *Network) Byzantine() []*Byzantine {
 	return net.byzantine
 }
 
+// Nodes returns all the nodes associated with the network.
+func (net *Network) Nodes() []*Node {
```

Comment:
Note that this doesn't return seed and sentry nodes (maybe it's worth mentioning this in comment)

Reply:
Updated.